openmls/src/extensions/mod.rs

```diff
@@ -514,7 +515,7 @@ mod test {
         // that all permutations keep their order after being (de)serialized.
         // The extension content does not matter in this test.
         let ext_x = Extension::ApplicationId(ApplicationIdExtension::new(b"Test"));
-        let ext_y = Extension::RatchetTree(RatchetTreeExtension::default());
+        let ext_y = Extension::ExternalPub(ExternalPubExtension::new(HpkePublicKey::new(vec![])));
```

Comment:
This change was required because `RatchetTree{,Extension}` should never have implemented `Default`.